hack/managed-kyma-migration/readme.md

```diff
@@ -103,10 +67,38 @@ adds the Istio module to the Kyma CR without overwriting existing module configu
 
 Perform the rollout to Stage together with the SRE team. Since they have already performed the rollout for other modules, they might suggest a different rollout strategy.
 
-1. Apply the module template to Stage Control Plane.
-2. `kcp login` to Stage, select some SKRs on `Kyma-Test/Kyma-Integration`, and run `migrate.sh` on them using `kcp taskrun`.
-3. Verify if the migration was successful on the SKRs by checking the status of Istio CR and the reconciler's components.
-4. Run `migrate.sh` for all SKRs in Kyma-Test and Kyma-Integration global accounts.
-5. Verify if the migration worked as expected.
-6. Run `migrate.sh` for the whole Canary landscape.
-7. Verify if the migration worked as expected.
\ No newline at end of file
+#### Prerequisites
+
+- Reconciliation is disabled for the Stage environment. See PR #4486 to the `kyma/management-plane-config` repository.
+
+#### Migration procedure
+1. Push the module to `experimental` channel in `kyma/module-manifests `repository.
+2. Test that experimental channeldeploys as expected by manually enabling it on a Stage managed cluster
+3. Apply the ModuleTemplate for both `fast` and `regular` channels to Stage Control Plane.
+4. Verify that the ModuleTemplate in the `fast` and `regular` channels is available in SAP BTP, Kyma runtime clusters of the Stage environment.
+5. Use `kcp login` to log in to Stage, select a few SAP BTP, Kyma runtime clusters on `Kyma-Test/Kyma-Integration`, and run `managed-kyma-migration.sh` on them using `kcp taskrun`.
```

Comment:
What do we do with the cluster from step 2? Shouldn't we change it to the fast channel?